exposures/configs/ksp-laravel-telescope-config-block.yaml

```diff
@@ -0,0 +1,20 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-laravel-telescope-config-block
+  namespace: default #Change to your namespace
+spec:
+  tags: ["laravel-telescope","exposure","telescope","config"]
+  message: "Alert! telescope request files is detected and blocked. Your sensitive files are secured."
+  selector:
+    matchLabels:
+      run: laravel-kubernetes-demo  #Change to labels of frontend web application pod
+  file:
+    severity: 6
+    matchPatterns:
+     - pattern: /**/telescope/requests
```

Comment:
@yasin-cs-ko-ak Can you check if it'll block the functionality of application

Reply:
@raviknox Let me verify that.